### PR TITLE
Hbi traduction EN de la fiche specification securité SNC

### PR DESCRIPTION
**Pull request by H4516022Y**
hello la team,

voici une traduction de la fiche en EN.

Merci pour la revue

### CODE REVIEW

File: pages/account_and_service_management/account_information/security-specification-privatecloud-SNC/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+Following the customer’s decommission of the service and prior to the removal of the hard drive from the rack, an erasing robot applies a secure data erasure procedure based on the NIST SP 800-88 r1 level ‘Purge’. In case of technical constraints or limitations on certain ranges of hard drives and when the level 'Purge' cannot be applied, the erase at the level 'Clear' will run.
+
+On an infrastructure qualified as SecNumCloud, the 'Destroy' level is applied after the disk has been extracted from the array.
+
+### 10.2 - Erasure of technical data
+
+Following the customer’s resignation from the service, OVHcloud releases the resources allocated to him and deletes the configurations made during service delivery
+
+
```
Comment (Jessica41) [inline pages/account_and_service_management/account_information/security-specification-privatecloud-SNC/guide.en-gb.md:181]: Hello, in the FR version there are 11 points, but the EN version only has 10, is that intentional?

Comment (H4516022Y) [reply pages/account_and_service_management/account_information/security-specification-privatecloud-SNC/guide.en-gb.md:181]: hello
this part should not be translated, it should be only in French  (it's required by new HDS version)

Comment (Jessica41) [reply pages/account_and_service_management/account_information/security-specification-privatecloud-SNC/guide.en-gb.md:181]: alright, thanks!

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @H4516022Y

**Pull request merged**